contract/src/constants.rs

```diff
@@ -55,6 +55,13 @@ pub const METADATA_CUSTOM_VALIDATED: &str = "metadata_custom_validated";
 pub const METADATA_CEP78: &str = "metadata_cep78";
 pub const METADATA_NFT721: &str = "metadata_nft721";
 pub const METADATA_RAW: &str = "metadata_raw";
+pub const ALL_TOKEN_OWNERS: &str = "all_token_owners";
+pub const FORWARD_TRACKER: &str = "forward_tracker";
+pub const REVERSE_TRACKER: &str = "reverse_tracker";
+pub const PAGE_DICTIONARY_PREFIX: &str = "page";
+pub const MAX_PAGE_NUMBER: &str = "max_page_number";
+pub const TOKEN_COUNT_AT_UPGRADE: &str = "token_count_at_upgrade";
```

Comment:
UNMATCHED_HASH_COUNT, and count down to 0

Reply:
Applied suggestion